examples/trip98.py

```diff
@@ -0,0 +1,72 @@
+import pytrip as pt
+import pytrip.tripexecuter as pte
+
+# first define some paths and other important paramters
+patient_name = "TST000000"
+ctx_path = "/home/bassler/Projects/shieldhit/res/TST000/TST000000.ctx"
+vdx_path = "/home/bassler/Projects/shieldhit/res/TST000/TST000000.vdx"
+voi_name = "GTV"
+working_directory = "/home/bassler/Projects/shieldhit/res/TST000/"
+
+ddd_dir = "/home/bassler/TRiP98/base/DATA/DDD/12C/RF3MM"
+spc_dir = "/home/bassler/TRiP98/base/DATA/SPC/12C/RF3MM"
+sis_path = "/home/bassler/TRiP98/base/DATA/SIS/12C.sis"
+hlut_path = "/home/bassler/TRiP98/base/DATA/HLUT/19990218.hlut"
+dedx_path = "/home/bassler/TRiP98/base/DATA/DEDX/20040607.dedx"
+
+my_couch_angle = 90.0
+my_gantry_angle = 10.0
+my_target_voi = "GTV"  # the name must exist in the .vdx file
+my_projectile = "C"
+
+# load CT cube
+my_ctx = pt.CtxCube()
+my_ctx.read(ctx_path)
+
+# load VOIs
+my_vdx = pt.VdxCube("", my_ctx)  # my_vdx is the object which will hold all volumes of interest and the meta information
+my_vdx.read(vdx_path)  # load the .vdx file
+print(my_vdx.get_voi_names())  # show us all VOIs found in the .vdx file
+
+# next pick a the proper VOI which we want to plan on
+target_voi_temp = my_vdx.get_voi_by_name(my_target_voi)  # Select the requested VOI from the VdxCube object
+target_voi = pte.Voi("GTV_VOI", target_voi_temp)  # for technical reasons the voi must be cast into a new Voi object
```

Comment:
Think we should add some comment that we know this is not good, and we hope to have some resolution for this in a future release.